tests/integration-tests/tests/create/test_create.py

```diff
@@ -117,6 +119,39 @@ def test_create_imds_secured(
     assert_cluster_imds_v2_requirement_status(region, cluster, status)
 
 
+# FIXME: Move in tests_update after update policy is changed
+@pytest.mark.usefixtures("instance", "scheduler")
+def test_create_disable_sudo_access_for_default_user(
+    region, os, pcluster_config_reader, clusters_factory, architecture, scheduler_commands_factory
+):
+    """
+    Verify that the cluster removes the Sudo access for Default user
+    in all the nodes of the Cluster if the DisableSudoAccessForDefaultUser is enabled.
+    """
+    disable_sudo_access_default_user = True
+    cluster_config = pcluster_config_reader(disable_sudo_access_default_user=disable_sudo_access_default_user)
+    cluster = clusters_factory(cluster_config, raise_on_error=True)
+
+    logging.info("Checking default user has disabled sudo access after cluster creation")
+    assert_head_node_is_running(region, cluster)
+    for node_type in NodeType:
+        assert_default_user_has_desired_sudo_access(
+            cluster, node_type, disable_sudo_access_default_user, region, scheduler_commands_factory
+        )
+    # logging.info("Updating Cluster to disable sudo access")
```

Comment:
Is there value in keeping this commented out code?

Reply:
Just for me when I want to test after changing the  update policy of the feature. Will remove it and just add it in my local